### PR TITLE
more adds

### PR DESCRIPTION
**Pull request by codelahiru**
some more additions to style and html?

### CODE REVIEW

File: CSScheatSheet/CSS/style.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
+*{
```
Comment (chammiherath) [inline CSScheatSheet/CSS/style.css:1]: fine

Comment (codelahiru) [reply CSScheatSheet/CSS/style.css:1]: thnx

**Comment by codelahiru** [comment]
okay

**Comment by chammiherath** [comment]
ok

**Comment by codelahiru** [comment]
agreed

**Pull request merged**